Korail-iOS/Korail-iOS/Screens/TicketReservationScene/TrainInquiryView/TrainInquiryViewCell.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  TrainInquiryViewCell.swift
+//  Korail-iOS
+//
+//  Created by 몽이 누나 on 2022/11/21.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class TrainInquiryViewCell: UITableViewCell {
+    
+
+    public var checkDummy : Int = 0
```

Comment:
보통 타입 명시해줄 때 : 의 왼쪽은 붙여서 쓰는게 관습입니다~!
```suggestion
    public var checkDummy: Int = 0
```